### ISSUE TITLE
Support for include_adult query parameter

### ISSUE THREAD

**Issue by g3rb3n** [xbmc/metadata.themoviedb.org.python#176]
Currently the scraper does not find adult content on TMBD. 

The TMDB API has an include_adult query parameters, which defaults to false. It would be nice to have a setting include_adult to support this functionality.

**Comment by KarellenX** [comment]
No, I don't think that will happen. **Easily** scraping adult titles has been excluded for family safety. There is always the possibility of scraping the wrong movie which happens to be an adult movie with some graphic posters and fanart.

Adding an easily accessible setting also makes it too easy.

Your options...

1. You will need to use a Parsing NFO file 
2. Use the Search options shown here... https://kodi.wiki/view/Add-on:The_Movie_Database_Python#Search 
3. Modify your local scraper to allow scraping adult titles.

**Comment by g3rb3n** [comment]
For restricting content we have many age rating systems in the world, which differ per country. It would be a nice feature to be able to restrict content based on the age of the viewer and a rating system of choice. There is already a feature request for this.

For kids I have a separate kodi box which should not show any content, movies, series, posters, fan art above their age. Which I need to do manually, since there is no support for age restrictions.

It would be really nice to have a feature that respects age restrictions on every level. But if we implement it in this way, we still don't have a scraper that actually respects age restrictions. And we are also not able to use the scraper for when we are above a certain age.

What would be wrong of having a too easy setting? Then I can just set it to false on the kids kodi box and switch it to trie on the kodi box that we use as adults.

**Comment by rmrector** [comment]
Adult content is kept at arms length, that change won't be merged into the default scraper.

**Comment by g3rb3n** [comment]
What is considered to be 'adult content' is subjective, culturally dependent and a matter of ongoing debate. The scraper currently does scrape a lot of 'adult content' which includes extreme violence, horror,  nudity and sex. According to the people of TMDB the 'adult' flag is meant to mean 'hardcore pornography'. But if a movie contains has one such scene, it should not be flagged, if it has a minimum of two such scenes it should be flagged. And if it contains a minimum of two scenes, but if the movie was originally released as regular movie and reviewed as movie by critics it is not flagged. So 9 Songs, Love, Q aka Desire and Bad Luck Banging, Loony Porn, Blue Movie (1971) and even Cannibal Holocaust do get scraped, but Blue Movie (1969) not. I fail to see the logic in this, but that is just my subjective opinion, yours might differ.

Thanks for all the hard work and neat code. It took me more time to figure out why Blue Movie (1969) did get scraped as Blue Movie (1971), than modifying the code.
